proto/batch_plan.proto

```diff
@@ -67,10 +67,15 @@ message SourceNode {
 }
 
 message IcebergScanNode {
+  enum IcebergScanType {
+    DATA_SCAN = 0;
+    EQUALITY_DELETE_SCAN = 1;
+  }
```

Comment:
https://protobuf.dev/programming-guides/style/#enums

Please change this according to the style guide.